templates/fungible-asset-template/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
-    "_fmt": "prettier 'frontend/**/*.ts'",
+    "_fmt": "npx prettier 'frontend/**/*.ts'",
```

Comment:
i think it is because you need to first run "npm i"

Reply:
i have run "npm i", but i think if you just run prettier without npx, that requires you to have prettier globally

Reply:
Interesting.. i dont have it globally and was able to run it with npm on this env

Reply:
No issue with `pnpm fmt` in sdk repo, maybe it's a npm thing?

Reply:
ill close the PR for now, can fix it if report by others